rules/sinks/third_parties/sdk/segment/java.yaml

```diff
@@ -4,10 +4,74 @@
 
 sinks:
 
-  - id: ThirdParties.SDK.Segment
-    name: Segment
+  - id: ThirdParties.SDK.Segment.Android
```

Comment:
remove this pattern `com[.]segment[.]analytics[.]java`

Reply:
removed